src/main/resources/com/puppycrawl/tools/checkstyle/grammar/java.g

```diff
@@ -805,7 +805,7 @@ implementsClause
 
                        |
                        t:typeSpec[false]  // method or variable declaration(s)
-                       (    IDENT  // the name of the method
+                       (    id:id  // the name of the method
```

Comment:
Same question here

Reply:
See above; if this also concerns the usage of "id" vs "i", I think the additional verbosity is helpful in the more complex rules.